From 0d99c835c83e3a37ee1c962cdf77ad8983b9bd70 Mon Sep 17 00:00:00 2001
From: Guangyao-Bai <guangyao.bai@intel.com>
Date: Fri, 16 Jul 2021 11:55:02 +0800
Subject: [PATCH] [Media Common] Fix MMD Device Init Fail issue

Fix MMD Device Init Fail issue:
Decompressed output corruption on the image
---
 .../media_interfaces_m12_adlp/media_interfaces_g12_adlp.cpp     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media_driver/media_interface/media_interfaces_m12_adlp/media_interfaces_g12_adlp.cpp b/media_driver/media_interface/media_interfaces_m12_adlp/media_interfaces_g12_adlp.cpp
index 5b6eccf5a4bc..2f2b50887b44 100644
--- a/media_driver/media_interface/media_interfaces_m12_adlp/media_interfaces_g12_adlp.cpp
+++ b/media_driver/media_interface/media_interfaces_m12_adlp/media_interfaces_g12_adlp.cpp
@@ -47,7 +47,7 @@ static bool adlpRegisteredMhw =
 #ifdef _MMC_SUPPORTED
 static bool adlsRegisteredMmd =
     MediaInterfacesFactory<MmdDevice>::
-    RegisterHal<MmdDeviceG12Tgllp>((uint32_t)IGFX_TIGERLAKE_LP);
+    RegisterHal<MmdDeviceG12Tgllp>((uint32_t)IGFX_ALDERLAKE_P);
 #endif
 
 #define PLATFORM_INTEL_ADLP   23
-- 
2.29.0

